.gitignore

```diff
@@ -11,6 +11,7 @@ test/**/test_output
 
 node_modules/
 npm-debug.log
+.yarn-lock
```

Comment:
Project-level gitignores should only contain project-related ignores; anything else should be in a global gitignore - otherwise you get every IDE's dotfiles polluting every repo.

If you wanted to ignore `yarn.lock` I'd understand, but `.yarn-lock` isn't a file that exists, as far as I'm aware.